### PR TITLE
Fix #2037: file formatting under multiplechoiceinput package

### LINKED ISSUE #2037: Fix file formatting under multiplechoiceinput package
**Describe the bug**
All the files, under the package `domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput` are not in the correct format. 

**Expected behavior**
File format should be follow from this file - [TextInputEqualsRuleClassifierProviderTest](https://github.com/oppia/oppia-android/blob/develop/domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt
)

**Correct Order**
Constant values
Dagger injection
Local variables
Test Before setup
Tests
Dagger Setup

### PR DESCRIPTION
**Pull request by Arjupta**
Fixes #2037 
Removed the one-line functions to segregate the initialisation logic in the starting of file

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by anandwana001 — CHANGES_REQUESTED** [review]
please check the reference again on how to update the file.

**Comment by Arjupta** [comment]
> please check the reference again on how to update the file.

@anandwana001 I compared it with the reference file you mentioned in the issue. It looked similarly structured to me, can you suggest specific changes that you found
Thanks

**Comment by anandwana001** [comment]
> > please check the reference again on how to update the file.
> 
> @anandwana001 I compared it with the reference file you mentioned in the issue. It looked similarly structured to me, can you suggest specific changes that you found
> Thanks

Please use `InteractionObjectTestBuilder` as mentioned in the reference file.

**Comment by Arjupta** [comment]
> > > please check the reference again on how to update the file.
> > 
> > 
> > @anandwana001 I compared it with the reference file you mentioned in the issue. It looked similarly structured to me, can you suggest specific changes that you found
> > Thanks
> 
> Please use `InteractionObjectTestBuilder` as mentioned in the reference file.

Have done those changes. PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt [hunk 1]
```diff
@@ -23,9 +23,9 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class MultipleChoiceInputEqualsRuleClassifierProviderTest {
-  private val NON_NEGATIVE_VALUE_0 = createNonNegativeInt(value = 0)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 1)
-  private val STRING_VALUE_2 = createString(value = "test")
+  private val NON_NEGATIVE_VALUE_0 = InteractionObjectTestBuilder.createNonNegativeInt(value = 0)
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: Leave an empty line above, same given in the reference.

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt [hunk 2]
```diff
@@ -23,9 +23,9 @@ import kotlin.test.fail
...
+  private val NON_NEGATIVE_VALUE_1 = InteractionObjectTestBuilder.createNonNegativeInt(value = 1)
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: Update the variable naming, following the reference here.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @ARJUPTA 
Left some comments.

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt (hunk 1 above)
Comment (Arjupta) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: Done

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt (hunk 2 above)
Comment (Arjupta) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: Done.
Have changed the variable names as per my knowledge, are they correct ?

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt [hunk 3]
```diff
@@ -23,9 +23,10 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class MultipleChoiceInputEqualsRuleClassifierProviderTest {
-  private val NON_NEGATIVE_VALUE_0 = createNonNegativeInt(value = 0)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 1)
-  private val STRING_VALUE_2 = createString(value = "test")
+
+  private val INT_VALUE_TEST_0 = InteractionObjectTestBuilder.createNonNegativeInt(value = 0)
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: ` NON_NEGATIVE_VALUE_TEST_0`

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt [hunk 4]
```diff
@@ -23,9 +23,10 @@ import kotlin.test.fail
...
+  private val INT_VALUE_TEST_1 = InteractionObjectTestBuilder.createNonNegativeInt(value = 1)
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: ` NON_NEGATIVE_VALUE_TEST_1`

**Review by anandwana001 — CHANGES_REQUESTED** [review]
nit changes. @ARJUPTA

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt (hunk 3 above)
Comment (Arjupta) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: done

File: domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt (hunk 4 above)
Comment (Arjupta) [reply domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt]: done

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @ARJUPTA

**Comment by rt4914** [comment]
@anandwana001 Please resolve the comments above and assign it back to me for merging.

**Comment by rt4914** [comment]
@aggarwalpulkit596 Please give respective points to @ARJUPTA . Thanks.

**Comment by aggarwalpulkit596** [comment]
give 5 points to @ARJUPTA

**Comment by aggarwalpulkit596** [comment]
give @ARJUPTA 5 points

**Pull request merged**